kolibri/plugins/management/assets/src/vue/user-page/user-create-modal.vue

```diff
@@ -6,28 +6,42 @@
     @enter="createNewUser"
     @cancel="close"
   >
-    <div>
+    <form @submit.prevent="createNewUser">
       <!-- Fields for the user to fill out -->
       <section class="user-fields">
-        <div class="user-field">
-          <label for="name">{{$tr('name')}}</label>
-          <input @focus="clearStatus" type="text" class="add-form" id="name" autocomplete="name"  autofocus="true" required v-model="full_name">
-        </div>
-
-        <div class="user-field">
-          <label for="username">{{$tr('username')}}</label>
-          <input @focus="clearStatus" type="text" class="add-form" autocomplete="username" id="username" required v-model="username">
-        </div>
-
-        <div class="user-field">
-          <label for="password">{{$tr('password')}}</label>
-          <input @focus="clearStatus" type="password" class="add-form" id="password" required v-model="password">
-        </div>
-
-        <div class="user-field">
-          <label for="confirm-password">{{$tr('confirmPassword')}}</label>
-          <input @focus="clearStatus" type="password" class="add-form" id="confirm-password" required v-model="passwordConfirm">
-        </div>
+        <core-textbox
+          :label="$tr('name')"
+          :autofocus="true"
+          @focus="clearStatus"
+          type="text"
+          class="user-field"
+          autocomplete="name"
+          required
+          v-model="full_name"/>
```

Comment:
But it's in the data object too: https://github.com/DXCanas/kolibri/blob/3c34b70d37ef9039ad0dd2bef9adc381ecd97d1d/kolibri/plugins/management/assets/src/vue/user-page/user-create-modal.vue#L109